db/migrate/20160525214823_promote_plant_parts_to_trait_descriptors.rb

```diff
@@ -0,0 +1,10 @@
+class PromotePlantPartsToTraitDescriptors < ActiveRecord::Migration
```

Comment:
@Nuanda It seems to me that this migration and the one preceding it are two halves of the same apple, i.e. they jointly accomplish an operation which is meant to be atomic. Accordingly, I would suggest merging this migration with `20160525172425_create_traits.rb`.